src/cplscheme/tests/ExplicitCouplingSchemeTest.cpp

```diff
@@ -80,7 +80,7 @@ void runSimpleExplicitCoupling(
     BOOST_TEST(computedTimesteps == 10);
     BOOST_TEST(not cplScheme.isActionRequired("constants::actionWriteIterationCheckpoint()"));
     BOOST_TEST(not cplScheme.isActionRequired("constants::actionReadIterationCheckpoint()"));
-    BOOST_TEST(cplScheme.isCouplingTimestepComplete());
+    BOOST_TEST(cplScheme.isTimeWindowComplete());
     BOOST_TEST(not cplScheme.isCouplingOngoing());
     BOOST_TEST(cplScheme.getNextTimestepMaxLength() > 0.0);
```

Comment:
I think that the name of this function is fine, isn't it? We do independent timesteps inside a window. This method returns the maximum allowed length for the timestep within the window. Therefore, the name is fine.